src/mscorlib/src/System/AggregateException.cs

```diff
@@ -474,23 +474,6 @@ public override string ToString()
 
             return text;
         }
-
-        /// <summary>
-        /// This helper property is used by the DebuggerDisplay.
-        /// 
-        /// Note that we don't want to remove this property and change the debugger display to {InnerExceptions.Count} 
-        /// because DebuggerDisplay should be a single property access or parameterless method call, so that the debugger 
-        /// can use a fast path without using the expression evaluator.
-        /// 
-        /// See http://msdn.microsoft.com/en-us/library/x810d419.aspx
-        /// </summary>
-        private int InnerExceptionCount
```

Comment:
Removing this will break the DebuggerDisplayAttribute (it should not have been removed by the rewriter).  See the above comment.